zebra-state/src/error.rs

```diff
@@ -74,4 +78,17 @@ pub enum ValidateContextError {
         difficulty_threshold: CompactDifficulty,
         expected_difficulty: CompactDifficulty,
     },
+
+    #[error("block contains an invalid commitment")]
+    InvalidCommitment(#[from] CommitmentError),
```

Comment:
We have many kinds of commitments, let's make it clear which one this refers to:

```suggestion
    InvalidBlockCommitment(#[from] block::CommitmentError),
```

Reply:
Done in 8dae6e73